src/bin/scan_target_groups.py

```diff
@@ -0,0 +1,184 @@
+    scan_target_group_id: UUID = typer.Argument(
+        ..., help="UUID of the scan target group"
+    ),
+):
+    """
+    Deletes the scan target group of the organization.
+    """
+    client = Client(profile=sdk_config.profile)
+    dump_json(
+        client.delete_organization_scan_target_group(
+            organization_id, scan_target_group_id
+        )
+    )
+
+
+@app.command(name="update")
+def scan_target_groups_update(
+    organization_id: UUID = typer.Argument(..., help="UUID of the organization"),
+    scan_target_group_id: UUID = typer.Argument(
+        ..., help="UUID of the scan target group"
+    ),
+    name: str = typer.Argument(..., help="new name of the scan target group"),
+):
+    """
+    Updates a scan target group.
+    """
+    client = Client(profile=sdk_config.profile)
+    dump_json(
+        client.update_scan_target_group(organization_id, scan_target_group_id, name)
+    )
+
+
+@app.command(name="create")
+def scan_target_groups_create(
+    organization_id: UUID = typer.Argument(..., help="UUID of the organization"),
+    kind: ScanTargetKind = typer.Argument(
+        ..., help="kind of the scan target group. Should be 'ORACLE'"
+    ),
+    name: str = typer.Argument(..., help="name of the scan target group"),
+):
+    """
+    Creates a scan target group for the organization.
+    """
+    client = Client(profile=sdk_config.profile)
+    dump_json(client.create_scan_target_group(organization_id, kind, name))
+
+
+@app.command(name="script")
+def scan_target_groups_script(
+    organization_id: UUID = typer.Argument(..., help="UUID of the organization"),
+    scan_target_group_id: UUID = typer.Argument(
+        ..., help="UUID of the scan target group"
+    ),
+):
+    """
+    Gets download URL of the scan target group.
+    """
+    client = Client(profile=sdk_config.profile)
+    dump_json(
+        client.get_scan_target_group_script(organization_id, scan_target_group_id)
+    )
+
+
+@app.command(name="compartments")
+def scan_target_groups_compartments(
+    organization_id: UUID = typer.Argument(..., help="UUID of the organization"),
+    scan_target_group_id: UUID = typer.Argument(
+        ..., help="UUID of the scan target group"
+    ),
+):
+    """
+    Iterates over the compartments of a scan target group.
+    """
+    client = Client(profile=sdk_config.profile)
+    output_iterable(
+        client.iter_scan_target_group_compartments(
+            organization_id, scan_target_group_id
+        )
+    )
+
+
+@app.command(name="insert")
+def scan_target_groups_insert(
+    organization_id: UUID = typer.Argument(..., help="UUID of the organization"),
+    scan_target_group_id: UUID = typer.Argument(
+        ..., help="UUID of the scan target group"
+    ),
+    region: str = typer.Argument(..., help="Oracle cloud region"),
+    tenancy_id: str = typer.Argument(..., help="Oracle tenancyId"),
+    user_id: str = typer.Argument(..., help="Oracle UserId"),
```

Comment:
Please double check whether it's UserId or userId like tenancyId. Or perhaps it's TenancyId. It seems strange as it is, check on the UI and let's standardize.